src/libpmemobj/lane.c

```diff
@@ -49,19 +49,19 @@
 #include "obj.h"
 #include "valgrind_internal.h"
 
-__thread int lane_idx = -1;
-static int next_lane_idx = 0;
+__thread int Lane_idx = -1;
+static int Next_lane_idx = 0;
 
-struct section_operations *section_ops[MAX_LANE_SECTION];
+struct section_operations *Section_ops[MAX_LANE_SECTION];
 
 /*
  * lane_get_layout -- (internal) calculates the real pointer of the lane layout
  */
 static struct lane_layout *
-lane_get_layout(PMEMobjpool *pop, int lane_idx)
+lane_get_layout(PMEMobjpool *pop, int lidx)
```

Comment:
Since the global variable name has changed, I don't think we need to modify this.